### PR TITLE
Add getOperationsDetails

### PR DESCRIPTION
**Pull request by mgavrila**
### Feature
Add getOperationsDetails in order to get visible operations and add the possibility to filter the array by `action` `sender` and `receiver`.

### Contains breaking changes
[x] No

[] Yes

### Updated CHANGELOG
[x] Yes

### Testing
[x] User testing
[] Unit tests

### CODE REVIEW

File: src/utils/transactions/getOperationsDetails.ts [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+
+export const getOperationsDetails = ({
+  transaction,
+  filterBy
+}: OperationDetailsPropsType): OperationType[] => {
+  if (!transaction.operations) {
+    return [];
+  }
+
+  let operations: OperationType[] = getVisibleOperations(transaction);
```
Comment (andreigiura) [inline src/utils/transactions/getOperationsDetails.ts:21]: Doesn't  getVisibleOperations return a OperationType[]?  is this needed let operations: **OperationType[]**

**Review by andreigiura — APPROVED** [review]

File: src/utils/transactions/getOperationsDetails.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
+import { InterpretedTransactionType, OperationType } from 'types';
+import { getVisibleOperations } from 'utils/transactions';
+
+type OperationDetailsPropsType = {
```
Comment (arhtudormorar) [inline src/utils/transactions/getOperationsDetails.ts:4]: export

File: src/utils/transactions/getOperationsDetails.ts [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    return [];
+  }
+
+  let operations = getVisibleOperations(transaction);
+
+  if (operations.length === 0) {
+    return [];
+  }
+
+  if (filterBy) {
```
Comment (arhtudormorar) [inline src/utils/transactions/getOperationsDetails.ts:27]: if !filterBy {
return operations

File: src/utils/transactions/getOperationsDetails.ts [hunk 4]
```diff
@@ -0,0 +1,48 @@
...
+type OperationDetailsPropsType = {
+  transaction: InterpretedTransactionType;
+  filterBy?: {
+    action?: OperationType['action'];
+    sender?: OperationType['sender'];
+    receiver?: OperationType['receiver'];
+  };
+};
+
+export const getOperationsDetails = ({
```
Comment (arhtudormorar) [inline src/utils/transactions/getOperationsDetails.ts:13]: add minimal unit test

**Review by radumojic — APPROVED** [review]

**Review by arhtudormorar — APPROVED** [review]

**Pull request merged**